protocol_test.go

```diff
@@ -87,7 +87,7 @@ func TestLongCall(t *testing.T) {
 
 	longResult, err := cli.LongCall(context.Background())
 	require.Nil(t, err, "call should have succeeded")
-	require.Equal(t, longResult, 100, "call should have succeeded")
+	require.Equal(t, longResult, 10, "call should have succeeded")
```

Comment:
This is just cleaner because it spins less and gives the context just as much time to cancel.